data/address/addressGenerator.ts

```diff
@@ -0,0 +1,22 @@
+import { DataGenerator } from "@data/generators";
+import { Address, AddressOptions, AddressIncludes } from "./address";
+import { fakerEN_GB as faker } from "@faker-js/faker";
+import { resolveIncludeAll } from "@data/utils";
+
+const allIncludes: AddressIncludes[] = ['address2', 'city', 'region', 'country']
```

Comment:
I wonder, would this be better served being statically defined in the accompanying .d file so that it sits along side where the keys are defined. Might this make it easier to remember to look at this in future if someone adds a new option in future.
Another thing to think about and experiment with I think.